packtools/sps/validation/article_license.py

```diff
@@ -0,0 +1,45 @@
+from packtools.sps.models.article_license import ArticleLicense
+
+
+class ArticleLicenseValidation:
+    def __init__(self, xmltree):
+        self.article_license = ArticleLicense(xmltree)
+
+    def validate_license(self, expected_value):
+        obtained_value = self.article_license.license_p
+        resp = {
+            "obtained_value": obtained_value,
+            "expected_value": expected_value
+        }
+        if obtained_value == expected_value:
+            resp.update({
+                "result": "ok"
+            })
+        else:
+            resp.update({
+                "result": "error",
+                "message": "the license text does not match the license text adopted by the journal"
+            })
+        return resp
+
+    def validate_license_code(self, expected_value):
```

Comment:
O código da licença é `by` e a versão é o valor após `/`

Reply:
A validação pode ser passar o código e a versão e ver se `code/version` in `link`

Reply:
OK, @robertatakenaka.

Reply:
Ok, @robertatakenaka.